src/components/create-application-form/index.tsx

```diff
@@ -247,6 +247,7 @@ export class CreateApplicationForm extends Component<
           <AppConfigAdGroups
             adGroups={applicationRegistration.adGroups}
             handleAdGroupsChange={this.handleAdGroupsChange}
+            labeling="Administrators"
```

Comment:
Should users be able to configure reader groups when registering new applications?